src/schema/alerts.ts

```diff
@@ -167,6 +173,19 @@ export const resolvers: IResolvers<any, Context> = traceResolvers({
       { data }: { data: GQLUpdateAlertsInput },
       ctx,
     ): Promise<GQLAlerts> => updateAlerts(ctx.con, ctx.userId, data),
+    updateLastReferralReminder: async (
+      _,
+      __,
+      ctx,
+    ): Promise<GQLEmptyResponse> => {
+      await updateAlerts<Partial<Alerts>>(ctx.con, ctx.userId, {
+        showGenericReferral: false,
+        flags: {
+          lastReferralReminder: new Date(),
+        },
```

Comment:
We need to make sure to use `updateFlagsStatement` because like this if `alert.flags` had any values already set it would overwrite.

Reply:
Ah yes, I was playing with some changes, must have accidentally removed it.
Let me revert back, had some issues with save usage